test/redirects.test.js

```diff
@@ -404,4 +405,68 @@ lab.experiment('hapi-redirect', () => {
       });
     });
   });
+
+  lab.test(' accepts a callback that adds additional dynamic routes', (done) => {
+    let count = 0;
+    server.register({
+      register: redirectModule,
+      options: {
+        log: true,
+        log404: true,
+        getRedirects(slug, redirectDone) {

```

Comment:
what is slug?

Reply:
are we able to do both function redirects and static redirects at the same time?  if so, can we add a test for it?

Reply:
Ping